content/docs/command-reference/cache/dir.md

```diff
@@ -50,17 +61,88 @@ for the config file.
 ## Example: Using relative path
 
 ```dvc
-$ dvc cache dir ../dir
+$ dvc cache dir ../dir $ cat .dvc/config
+...
+[cache] dir = ../../dir
+...
+```
+
+`../dir` has been resolved relative to the `.dvc/` dir, resulting in
+`../../dir`.
+
+Thecommand `dvc cache dir` resolves your relative paths so that they work with
+the location of the config file in your project.
+
+## Example: Using absolute path
+
+```dvc
```

Comment:
Again here. I don't know what's going on. The code block isn't properly closed. Please run the web app locally to see how this would look like. (The instructions are in the docs contrib guide.)

Reply:
Wait never mind this part is OK I was confused... I meant a different section, let me delete this comment and make it again in the right place.